### PR TITLE
fix: update strong type

### PR DESCRIPTION
**Pull request by joyqvq**
## Description 

updates to include: https://github.com/MystenLabs/fastcrypto/pull/751

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-open-rpc/spec/openrpc.json [hunk 1]
```diff
@@ -5317,6 +5317,28 @@
...
+        "description": "A struct that stores a Bn254 Fr field element as 32 bytes.",
+        "type": "array",
+        "items": {
+          "type": "integer",
+          "format": "uint8",
+          "minimum": 0.0
+        },
+        "maxItems": 32,
+        "minItems": 32
+      },
```
Comment (bmwill) [inline crates/sui-open-rpc/spec/openrpc.json:5341]: This doesn't look right. The JsonSchema should match the serialization format which is a String and not an array of [u8; 32]

**Review by bmwill — COMMENTED** [review]
looks good except the changes to the openrpc spec

**Comment by joyqvq** [comment]
> looks good except the changes to the openrpc spec

first time looking at this, do you know much how the jsonschema trait works? is it derived from the serialize trait?

the test for an serialized zklogin signature itself suggests that the format didn't change

**Comment by bmwill** [comment]
> > looks good except the changes to the openrpc spec
> 
> first time looking at this, do you know much how the jsonschema trait works? is it derived from the serialize trait?
> 
> the test for a fixed serialized zklogin signature itself still passes, i wonder if the format indeed didn't change

I don't know how that works, but afaik its not derived from the serialization trait, but rather done in parallel with it assuming you just used serde_derive to do the ser/de impls. The format won't have changed but this spec is used from documentation and possibly for generating client impls in other languages

**Comment by joyqvq** [comment]
> > > looks good except the changes to the openrpc spec
> > 
> > 
> > first time looking at this, do you know much how the jsonschema trait works? is it derived from the serialize trait?
> > the test for a fixed serialized zklogin signature itself still passes, i wonder if the format indeed didn't change
> 
> I don't know how that works, but afaik its not derived from the serialization trait, but rather done in parallel with it assuming you just used serde_derive to do the ser/de impls. The format won't have changed but this spec is used from documentation and possibly for generating client impls in other languages

Revised in fastcrypto: https://github.com/MystenLabs/fastcrypto/pull/754 PTAL

**Review by bmwill — APPROVED** [review]

**Pull request merged**